src/data/markdown/translated-guides/en/03 Results output/200 Real-time.md

```diff
@@ -34,7 +34,7 @@ You can also stream real-time metrics to the following services:
 - [InfluxDB](/results-output/real-time/influxdb-grafana)
 - [Netdata](/results-output/real-time/netdata)
 - [New Relic](/results-output/real-time/new-relic)
-- [Prometheus](/results-output/real-time/prometheus)
+- [Prometheus remote write](/results-output/real-time/prometheus-remote-write)
```

Comment:
@MattDodsonEnglish this link does not work with the current change. Can I rename the file or could it break something?

Reply:
Rename the file is fine, but I don't think we need to rename the page path.

On the remote write page, I added a `slug` in the frontmatter so the path stays the same. The PR passed without any link breaks:

https://github.com/grafana/k6-docs/actions/runs/3714477284/jobs/6298652654

Reply:
If I try it on staging it redirects to the home page. Is it an issue with staging?

Reply:
ohhh. Yes, it's a problem. I think all 404s have started redirecting to the home page, which means the linkchecker is useless :-( .

Reply:
Besides the linkchecker problem, this still goes to the right page for me. Am I missing something?
https://staging.k6.io/docs/results-output/real-time/prometheus/

Reply:
@MattDodsonEnglish The main issue is that it is only `/prometheus` and we are trying to set everything as Prometheus **remote write**. It seems not consistent to me. WDYT?

Reply:
I see. I've been thinking that Prometheus Remote Write is the only viable export from k6 to Prometheus. Do you think k6 might export to more Prometheus services in the future?

If there will be more, we have two options:
 - Add a new item in the "Real time" list:
   ```
   RESULTS OUTPUT
  Real time >
    AWS
    Grafana
    Prometheus Remote Write
    Prometheus hypothetical
  ```
- Add a nested list
 ``` 
  Real time >
    AWS
    Grafana
    Prometheus >
      - Remote Write
      - Prometheus hypothetical
  ```
  
  Even thought it's not great to have so much nesting, the second option seem most logical. What's more, if we change the URL to `prometheus-rw` now, we'll have to redirect `prometheus` to that URI, creating another set of problems. I'm going to merge, but let this be a record of my decision-making process.

Reply:
Finally, if we do come into a situation with some messy redirects to figure out, we can at last come up with a "clean" set of URIs using `prom*` .

Reply:
The Prometheus remote write is a protocol and it has many implementations the Prometheus server itself is one of them. Remote write alone is not correct, I see it creates a bunch of confusion. Naming is a difficult thing in IT world. :smile:

Reply:
Oh, that is annoying! I didn't think of it this way. I'll change the URI to `prometheus-rw` and program in some redirects. 

Why isn't `remote write` correct on its own? Just wondering?

Reply:
I guess it's incorrect because that's just not the name of the protocol.

Reply:
>I guess it's incorrect because that's just not the name of the protocol.

Exactly. There isn't any official explanation, but this is the title in the spec, so this is my personal interpretation.